shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-suite/src/test/java/org/apache/shardingsphere/test/integration/cases/assertion/IntegrationTestCase.java

```diff
@@ -35,6 +37,9 @@
 @XmlAccessorType(XmlAccessType.FIELD)
 public final class IntegrationTestCase {
     
+    private static final String DB_NAME = "\\{dbName}";
```

Comment:
This is pojo for jaxb to convert XML only, could you move the business logic to other object?